tests/test_playbooks/content_rhel_role.yml

```
@@ -0,0 +1,35 @@
+---
+- hosts: localhost
+  collections:
+    - theforeman.foreman
+  gather_facts: false
+  vars_files:
+    - vars/server.yml
+  tasks:
+    - name: ensure test organization
+      include_tasks: tasks/organization.yml
+      vars:
+        organization_name: "Test Organization"
+        organization_state: "present"
+
+- hosts: tests
+  collections:
+    - theforeman.foreman
+  gather_facts: false
+  vars_files:
+    - vars/server.yml
+  roles:
+    - role: content_rhel
+      vars:
+        foreman_organization: "Test Organization"
+        username: "{{ foreman_username }}"
+        password: "{{ foreman_password }}"
+        server_url: "{{ foreman_server_url }}"
+        validate_certs: false
+        manifest_download: false
+        foreman_manifest_path: "{{ subscription_manifest_path }}"
+        foreman_sync_plan_name: "RHEL Sync Plan"
+        foreman_sync_plan_interval: daily
+        foreman_sync_plan_sync_date: 2021-02-02 00:00:00 UTC
+        foreman_activation_key_name: "basic_rhel_key"
+        foreman_content_rhel_wait_for_syncs: true
```

```suggestion
        foreman_content_rhel_wait_for_syncs: "{{ ansible_version['full'] is version('2.9', '>=') }}"
```

that should allow you to test unsync on 2.8, as discussed.

see https://docs.ansible.com/ansible/latest/user_guide/playbooks_tests.html#comparing-versions for details about `version` test and https://docs.ansible.com/ansible/latest/reference_appendices/special_variables.html for the `ansible_version` magic var

---

Ah, this is a much nicer approach than checking in tests/test_crud.py

Thanks!